addons/calendar/models/calendar_event.py

```diff
@@ -73,6 +73,11 @@ class Meeting(models.Model):
 
     DISCUSS_ROUTE = 'calendar/join_videocall'
 
+    @api.model
+    def get_activity_excluded_models(self):
```

Comment:
Would make this private and move it to the "TOOLS" section of the file.

"Return a list of models where we don't want to create activities."

->

> For some models, we don't want to automatically create activities when a calendar.event is created.
> (This is the case notably for appointment.types)
> This hook method allows to specify those models.
> See calendar.event create method for details.

(Sorry, I just noticed this change now, wasn't notified on the PR yet).